spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -369,12 +369,26 @@ of the graph is a task application.  This is done by using the DSL for composed
 tasks.  A composed task can be created via the RESTful API, the Spring Cloud
 Data Flow Shell, or the Spring Cloud Data Flow UI.
 
+=== Configuring Composed Task in Spring Cloud Data Flow
+Composed tasks are executed via a task application called the https://github.com/spring-cloud-task-app-starters/composed-task-runner[Composed Task Runner].
+Out of the box this application is not registered with Spring Cloud Data Flow
+by default.  So to create composed tasks we must first register the Composed
+Task Runner as an application with Spring Cloud Data Flow as follows:
+```
+app register --name composed-task-runner --type task --uri maven://org.springframework.cloud.task.app:composedtaskrunner-task:1.2.0.RELEASE
```

Comment:
Is this the correct version? we are currently using 1.0.0.BUILD-SNAPSHOT, also, should this be a placeholder?